internal/app/secretless/handlers/ssl/ssl.go

```diff
@@ -9,14 +9,14 @@ import (
 
 type options map[string]string
 
-type SecretlessTLSConfig struct {
+type DbSSLMode struct {
```

Comment:
what's the thought behind this change? it seems like this struct contains more than just the ssl mode, esp if I think of ssl mode as the string parameter from the secretless config that defines how to handle SSL in the secretless -> backend connection. I would have thought `DbTLSConfig` was sufficient

Reply:
Your instinct here is spot on.  The problem is that it's the wrong abstraction, and it's doing multiple things, so every name will be somewhat wrong.  This will be fixed during the ssl refactoring issue, already in waffle.

For now I do think this is slightly "less wrong" than `DbTLSConfig` because if you look at the logic around this, most of it has to do with ssl mode stuff, even though it also has the certs themselves.  so i'd like to label this `WONT_FIX_UNTIL_SSL_TICKET`.